### PR TITLE
HubSpot integration fixes and improvements

### PR DESCRIPTION
**Pull request by mbmjertan**
# Description

This PR:
* changes the `HubspotClient::getApiKey` method conditional so we return API keys properly
* changes the `HubspotClient::getItems` method to return `submitText` as well
* changes the `Hubspot::getFormFields` and `getFields` methods to consume the whole `$item` array so `submitText` can be used (without mapping it to a field manually in `HubspotClient`)

### CODE REVIEW

**Review by iruzevic — CHANGES_REQUESTED** [review]
I need to check this before the merge I'm not sure in this implementation

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -13,76 +13,103 @@ on:
     paths-ignore:
       - '**.md'
 
-# Cancels all previous workflow runs for the same branch that have not yet completed.
```
Comment (dingo-d) [inline .github/workflows/ci.yml:16]: Leave the concurrency in 🙏🏼

**Review by iruzevic — APPROVED** [review]
I have tested this and it is ok. tnx

File: .github/workflows/ci.yml (hunk 1 above)
Comment (mbmjertan) [reply .github/workflows/ci.yml:16]: @dingo-d, sure, but I don't know how to fix these CI issues. I can remove these changes entirely, this was just testing to see how to fix CI.

Unfortunately this is still stuck on Composer failing to authenticate against GitHub.

**Comment by dingo-d** [comment]
@mbmjertan I've updated some composer packages, so phpstan will probably fail. Also, I may have stumbled upon another edge case for the coding standards so I'll have to fix those, which is why I constrained the standards to v1.3.

Should be ok now.

**Review by dingo-d — APPROVED** [review]

**Pull request merged**